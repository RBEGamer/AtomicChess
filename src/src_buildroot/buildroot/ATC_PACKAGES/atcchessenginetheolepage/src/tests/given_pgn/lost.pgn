[Event "?"]
[Site "Portugal"]
[Date "1941.??.??"]
[Round "?"]
[White "Alekhine, Alexander A"]
[Black "Cruz"]
[Result "1-0"]

1. e2-e4 e7-e5 2. Ng1-f3 Ng8-f6 3. d2-d4 e5xd4 4. e4-e5 Nf6-d5 5. Qd1xd4
Nd5-b4 6. Nb1-a3 Nb4-c6 7. Qd4-f4 Bf8-e7 8. Bf1-d3 *

