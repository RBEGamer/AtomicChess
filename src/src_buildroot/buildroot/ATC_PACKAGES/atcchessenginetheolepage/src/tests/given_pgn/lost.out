[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 4 rank: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[piece_taken]	 piece: 5 file: 4 rank: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 3
[move]	board::PieceType: 3
